californium-core/src/main/java/org/eclipse/californium/core/network/stack/BlockwiseLayer.java

```diff
@@ -925,8 +928,9 @@ private void handleBlock2Response(final Exchange exchange, final Response respon
 			} else {
 				// ERROR, wrong block number (server error)
 				// Canceling the request would interfere with Observe, so just ignore it
-				LOGGER.warn("ignoring block2 response with wrong block number {} (expected {}): {}", block2.getNum(),
-						status.getCurrentNum(), response);
+				ignoredBlock2.incrementAndGet();
```

Comment:
I don't know if we should increment only if healthlogger is activated.

Reply:
Currently it*s not more triggered ... at least at my machine :-).